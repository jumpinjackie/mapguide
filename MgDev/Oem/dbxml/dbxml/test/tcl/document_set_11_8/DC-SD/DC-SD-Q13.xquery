for $a in collection("11.8.2.dbxml")/catalog/item
where $a/date_of_release gt "1990-01-01" and
           $a/date_of_release lt "1991-01-01" and
           empty($a/publisher/contact_information/FAX_number)
return
      <Output>
        {$a/publisher/name}
      </Output>
