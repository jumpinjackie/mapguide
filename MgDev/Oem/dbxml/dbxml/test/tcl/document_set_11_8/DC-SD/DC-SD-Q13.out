<Output><name>beans within the dogged, bold dolphins can print fi</name></Output>